src/resolvers/auth.js

```diff
@@ -41,15 +47,11 @@ const authResolvers = {
           return null;
         }
 
-        const refreshToken = jwt.sign({ userId: user.id }, REFRESH_TOKEN_SECRET, {
-          expiresIn: '7d',
-        });
-        const accessToken = jwt.sign({ userId: user.id }, ACCESS_TOKEN_SECRET, {
-          expiresIn: '15min',
-        });
+        const refreshToken = createNewRefreshToken(user.id);
+        const accessToken = createNewAccessToken(user.id);
 
-        res.cookie('refresh-token', refreshToken);
-        res.cookie('access-token', accessToken);
+        addAccessTokenCookie(res, accessToken);
+        addRefreshTokenCookie(res, refreshToken);
```

Comment:
awesome! last request: let's add these two cookies to the `register` mutation as well!